student/migrations/0036_auto_20210410_2112.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-04-10 21:12
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('advising', '0001_initial'),
+        ('timetable', '0033_section_course_section_id'),
+        ('student', '0035_auto_20210323_2126'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='student',
+            name='advisors',
+            field=models.ManyToManyField(related_name='students', to='advising.Advisor'),
+        ),
+        migrations.AddField(
+            model_name='student',
+            name='minors',
+            field=django.contrib.postgres.fields.ArrayField(base_field=models.CharField(default=b'', max_length=255), default=list, size=None),
+        ),
+        migrations.AddField(
+            model_name='student',
+            name='other_majors',
+            field=django.contrib.postgres.fields.ArrayField(base_field=models.CharField(default=b'', max_length=255), default=list, size=None),
+        ),
+        migrations.AddField(
+            model_name='student',
+            name='primary_major',
+            field=models.CharField(default=b'', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='student',
+            name='sections',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='timetable.Section'),
```

Comment:
since you changed the name of this field in models you should change it here to avoid two migration files - or delete this file and re-migrate